extensions/kafka-client/runtime/src/main/java/io/quarkus/kafka/client/runtime/KafkaRuntimeConfigProducer.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.kafka.client.runtime;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.StreamSupport;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Produces;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import io.quarkus.arc.DefaultBean;
+
+@Dependent
+public class KafkaRuntimeConfigProducer {
+
+    private String configPrefix = "kafka";
```

Comment:
If I understand correctly you copy the "kafka" configuration and configure the health check admin client with them.
Is it really related to this PR?

Reply:
I'd say so. The admin client needs the same security configuration as all consumers/producers. Otherwise authentication will fail once the broker is set up to expect it.

Reply:
Yes and no. You have applications using multiple brokers, and some parts of Quarkus use a different configuration format.